.github/workflows/main.yml

```diff
@@ -334,10 +336,3 @@ jobs:
           CI: true
         run: yarn test-external:${{ matrix.partner }}
         continue-on-error: ${{ matrix['continue-on-error'] == true }}
-      - name: upload built assets
-        if: failure() || success()
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ matrix.partner }}-dist
-          path: ${{ github.workspace }}/__external-test-cache/${{ matrix.partner }}/dist/assets
-        continue-on-error: ${{ matrix['continue-on-error'] == true }}
```

Comment:
We probably don't need this.  We can just run this locally

Reply:
I think the idea was it makes debugging faster if I understood correctly, no?

Reply:
Likely.  We do have those scripts `test-external:*` to run the specific failures and debug just as easily as well.